frontend/src/app/globals.css

```diff
@@ -196,6 +196,15 @@ body {
   .custom-table-head th {
     @apply text-[16px] leading-3 font-normal h-10;
   }
+
+  .upload-file-box {
```

Comment:
I suggest creating separate CSS files for each module instead of using this class throughout the application.